### ISSUE TITLE
Slack not working at all

### ISSUE THREAD

**Issue by peyidev** [meetfranz/recipe-slack#27]
Slack for me and my team stoped working a few hours back, we can't use it with Franz.

**Comment by tzikis** [comment]
Yeah, me neither. It says it's using an old browser, "This browser is no longer supported"

**Comment by Multisaft7** [comment]
Same here on my side.  But when I think back, the reference to the outdated browser and the shutdown was months ago.

**Comment by exedriver** [comment]
Pretty sure this is related to the "outdated" version of electron, Franz is based on. In the Franz 5.10 changelog it says "Update electron to 22.3.18", which uses Chromium v108 (https://releases.electronjs.org/release/v22.3.18).

Slacks new minimum Chrome version is 117 or higher (https://slack.com/intl/de-de/help/articles/115002037526-Systemanforderungen-f%C3%BCr-die-Nutzung-von-Slack), which means that the Electron version of Franz needs to be updated to match these requirements.

**Comment by peyidev** [comment]
I found Rambox, similar with more integrations.

https://rambox.app/

**Comment by iv-emehanika** [comment]
The same story. Is the project alive or is it running in slow-poke mode?

**Comment by EricMoerthVis** [comment]
Same problem on my side

**Comment by alexgoddity** [comment]
We’re very sorry but your browser is not supported!
same one.

**Comment by Marksson** [comment]
I switched to Rambox too - Thanks @peyidev! 

I have used Franz since 2016 and later upgraded to the paid version. It worked mostly fine but I remember that I had issues with Skype and just could not find any way to contact their support staff. That's not right imho, especially when you're a paid customer. So bye Franz, hello Rambox!

**Comment by dbuenoparedes** [comment]
Same problem here, seems Franz is dead, it was good while it lasted.

I also switched to Rambox as @peyidev suggested and I'm liking it so far, thanks for the suggestion!

**Comment by kazezz** [comment]
Same here, not working anymore @dbuenoparedes rambox is free?

**Comment by dbuenoparedes** [comment]
@kazezz it has a free tier/plan called "Basic", you can check it out here: https://rambox.app/pricing/. After you click on sign up and create your account you can start using it, to download you can do that here: https://rambox.app/download (although it seems broken as of now to me).

So far is very similar to Franz, feels a bit _heavier_ though.
